airflow/models/all_models.py

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Airflow models"""
+from airflow.models.base import ID_LEN, Base  # noqa: F401 # pylint: disable=unused-import
```

Comment:
Please remove the comments and add this file to `per-file-ignores` in .flake8. Can we use same approach for pylint?

Reply:
Agree with per-file-ignores. For pylint we should put  pylint: disable as the first line of the file - then it will be ignored for the whole file